canonical/ubuntu.com
# /desktop refresh

## Done

- Applied refresh as per [doc](https://docs.google.com/document/d/1ZhMkLlkXcimHTIe1taXy24vbbR9pBkZd49H2sJqeQH0/edit#heading=h.fqpazxhydeus) and [design](https://www.figma.com/file/wLOtPeQhAiExhQCwKYhi3H/24.04-U.com---desktop-(rebranding)?node-id=613%3A501&mode=dev)
- Applied /desktop/features -> /desktop redirect as per ticket
- Bumped Vanilla version

## QA

- View the site locally in your web browser at: https://ubuntu-com-13757.demos.haus/desktop
    - Be sure to test on mobile, tablet and desktop screen sizes
- Check against doc and design 

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-9643
